docs/dev/adding_instruments.rst

```diff
@@ -105,6 +105,13 @@ This is a minimal instrument definition:
 
 Make sure to include the PyMeasure license to each file, and add yourself as an author to the :code:`AUTHORS.txt` file.
```

Comment:
```suggestion
Make sure to include the PyMeasure license to each file, and add yourself as an author to the :code:`AUTHORS.txt` and :code:`docs/about/authors.rst` files.
```
By chance, I saw this one as well.

Reply:
I think I'll see if I can eliminate this nowadays needless duplication in this PR, instead.

Reply:
That would be even better. Thanks.

Reply:
This is now much easier than when I tried this last time. Not super beautiful because it's a literal block instead of a bullet list, but good enough imo.